go.mod

```diff
@@ -23,9 +23,9 @@ require (
 	k8s.io/apimachinery v0.27.1
 	k8s.io/client-go v0.27.1
 	k8s.io/cloud-provider v0.24.0
-	k8s.io/klog/v2 v2.90.1
-	k8s.io/utils v0.0.0-20230313181309-38a27ef9d749
-	sigs.k8s.io/controller-runtime v0.8.3
```

Comment:
can you please give us -- either here or in the commit message -- what exact commands you used to perform this change?

Reply:
I have no problem updating the commit message if that's proffered, but this was simply:
`go get -u controller-runtime; go mod tidy; go mod vendor;`